### PR TITLE
Indexing

### PR DESCRIPTION
**Pull request by Stanislava88**
Here i have solved five task: increase speed retrieval table using the index.

Analysis of the speed of retrieval without indexes
withouindex

cost to get all rows: 26354.00
total execution time in milliseconds : 86.195

Analysis of the speed of retrieval with indexes

withindex

cost to get all rows: 8.45
total execution time in milliseconds :0.062

### CODE REVIEW

File: src/com/clouway/index/Demo.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import java.sql.SQLException;
+
+/**
+ * @author Stanislava Kaukova(sisiivanovva@gmail.com)
+ */
+public class Demo {
+  public static void main(String[] args) {
+    try (Connection connection = DriverManager.getConnection("jdbc:postgresql://localhost/store", "postgres", "clouway.com");
+
+         PreparedStatement preparedStatement = connection.prepareStatement("INSERT INTO customer(id,firstName,lastName,egn,phoneNumber) VALUES (?,?,?,?,?)")) {
```
Comment (freekman) [inline src/com/clouway/index/Demo.java]: format

**Comment by freekman** [comment]
Write to me the select statements that you have used

**Comment by Stanislava88** [comment]
``` store=# explain (analyze) select firstName,lastName,phoneNumber from customer where egn='egn10'and id=10; ```

**Comment by freekman** [comment]
LGTM

**Pull request closed without merging**